cursive-core/src/logger.rs

```diff
@@ -44,12 +137,18 @@ pub fn log(record: &log::Record) {
 }
 
 impl log::Log for CursiveLogger {
-    fn enabled(&self, _metadata: &log::Metadata) -> bool {
-        true
+    fn enabled(&self, metadata: &log::Metadata) -> bool {
+        if metadata.target().contains("cursive_core") {
```

Comment:
Maybe we should be more restrictive, requiring this to be a prefix of the target?

Reply:
Done.